lib/mix/test/mix/tasks/test_test.exs

```diff
@@ -46,11 +52,11 @@ defmodule Mix.Tasks.TestTest do
   defp ex_unit_opts_from_given(passed) do
     passed
     |> ex_unit_opts()
-    |> Keyword.drop([:failures_manifest_file, :autorun])
+    |> Keyword.drop(@default_opts)
   end
 
   test "--stale: runs all tests for first run, then none on second" do
-    in_fixture "test_stale", fn ->
+    in_fixture"test_stale", fn ->
```

Comment:
Is the removal of the white space intentional?

Reply:
Not intentional